### PR TITLE
Implement component product images

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component product images

image

image

stoybook : https://6684faab2d0c99a3e93bf145-cbpyuacvbl.chromatic.com/

### CODE REVIEW

File: src/components/ProductImages/ProductImages.tsx [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+  const handleImageClick = (index: number) => {
+    setActiveIndex(index);
+  };
+
+  return (
+    <div>
+      <img
+        className={styles.detailImage}
+        src={images ? images[activeIndex] : ''}
+        alt={images[activeIndex]}
```
Comment (hungdo-agilityio) [inline src/components/ProductImages/ProductImages.tsx]: images is an array so there is any cases for this go in else statement?

File: src/components/ProductImages/ProductImages.tsx [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
+      />
+      <ul className={styles.imageGroup}>
+        {images.map((image, index) => (
+          <li key={index}>
```
Comment (hungdo-agilityio) [inline src/components/ProductImages/ProductImages.tsx]: recheck

File: src/components/ProductImages/ProductImages.tsx [hunk 3]
```diff
@@ -1,9 +1,38 @@
...
+            <img
+              className={`${styles.image} ${index === activeIndex ? styles.imageActive : ''}`}
+              src={image}
+              alt={`Product ${index + 1}`}
```
Comment (hungdo-agilityio) [inline src/components/ProductImages/ProductImages.tsx]: alt not good

File: src/components/ProductImages/ProductImages.stories.tsx [hunk 4]
```diff
@@ -1,5 +1,9 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import ProductImages from './ProductImages';
+import image1 from '../../assets/images/detail-small.jpg';
+import image2 from '../../assets/images/small-t-shirt.jpg';
+
+const images = [image1, image2, image1];
```
Comment (hungdo-agilityio) [inline src/components/ProductImages/ProductImages.stories.tsx]: provide me image for case less/more than 3 images

Comment (thanhtrantruong47) [reply src/components/ProductImages/ProductImages.stories.tsx]: less than 3 images

image

File: src/components/ProductImages/ProductImages.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/ProductImages/ProductImages.tsx]: Fixed

File: src/components/ProductImages/ProductImages.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply src/components/ProductImages/ProductImages.tsx]: Fixed

File: src/components/ProductImages/ProductImages.tsx (hunk 3 above)
Comment (thanhtrantruong47) [reply src/components/ProductImages/ProductImages.tsx]: Fixed

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**